ocs_ci/ocs/registry.py

```diff
@@ -367,19 +367,22 @@ def image_pull_and_push(project_name, template, image="", pattern="", wait=True)
         if f'"{template}" not found' in str(cfe):
             logger.warn(f"Template {template} not found")
             template = "redis-ephemeral"
+            image = "registry.redhat.io/rhel8/redis"
         else:
             raise
 
     if config.DEPLOYMENT.get("disconnected"):
         mirror_image(image=image)
     else:
-        cmd = f"new-app --template={template} -n {project_name}"
+        cmd = f"new-app --template={template} -n {project_name} --name=build1"
         ocp_obj = ocp.OCP()
         ocp_obj.exec_oc_cmd(command=cmd, out_yaml_format=False)
 
         # Validate it completed
         if wait:
             if template == "redis-ephemeral":
+                wait_time = 180
```

Comment:
Can we use wait for resource ? why 180 sec?

Reply:
no, we need the deploy/build name